python/tvm/micro/model_library_format.py

```diff
@@ -287,6 +288,9 @@ def reset(tarinfo):
             return tarinfo
 
         tar_f.add(str(source_dir), arcname=".", filter=reset)
+        is_aot = isinstance(mod, executor_factory.AOTExecutorFactoryModule)
+        if is_aot and str(mod.runtime) == "crt":
+            tar_f.add(get_standalone_crt_dir(), arcname="./runtime")
```

Comment:
Can we convert this `./runtime` to be a a variable defined somewhere. So that we can also use it everywhere we need this path identifier, for example, in the `standalone_crt = { ...` definition below around line `327`?